h2o-core/src/main/java/hex/ModelMetricsBinomialGLMGeneric.java

```diff
@@ -0,0 +1,30 @@
+package hex;
+
+import water.fvec.Frame;
+import water.util.TwoDimTable;
+
+public class ModelMetricsBinomialGLMGeneric extends ModelMetricsBinomialGeneric {
+
+  public final long _nullDegressOfFreedom;
+  public final long _residualDegressOfFreedom;
+  public final double _resDev;
+  public final double _nullDev;
+  public final double _AIC;
+  public final TwoDimTable _coefficients_table;
+
+  public ModelMetricsBinomialGLMGeneric(Model model, Frame frame, long nobs, double mse, String[] domain,
+                                        double sigma, AUC2 auc, double logloss, TwoDimTable gainsLiftTable,
+                                        CustomMetric customMetric, double mean_per_class_error, TwoDimTable thresholds_and_metric_scores,
```

Comment:
add pr_auc?

Reply:
Same answer as above.



It is and we do have it. It's present in the MojoModelMetricsBinomial and it is filled few lines above this line:

                auc._auc = binomial._auc;
                auc._pr_auc = binomial._pr_auc;
                auc._gini = binomial._gini;

because this part is common for both GLM binomial and "normal" binomial.